MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -76,12 +77,11 @@ object VisionManager : BaseVisionManager {
     private lateinit var sessionManager: SessionManager
     private lateinit var videoRecorder: VideoRecorder
 
-    private var isRecording = false
+    private lateinit var mapboxTelemetry: MapboxTelemetry
 
     private val handlerMain = Handler(Looper.getMainLooper())
 
-    // TODO temporary fix, this should be moved to core
-    private var currentCountry: Country = Country.Unknown
+    private var isTurnstileEventSent = false
```

Comment:
@RingerJK could you clarify we still need to send turnstile events?

Reply:
yep, analytic-team use it for audit and some other issues